tests/integration/reporter.spec.js

```diff
@@ -16,8 +16,8 @@ describe('runs tests on cloud', function () {
     expect(process.env.SAUCE_ACCESS_KEY).toBeDefined();
 
     const cucumberRunCommand = 'npx cucumber-js'
-    const format = path.join(process.cwd(), 'src/reporter.ts')
```

Comment:
Why was this working before, but not anymore? 🤔

Reply:
I'm also confused about this change.